app/Http/Controllers/earhquakescontroller.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\earthquakes;
+use App\daydata;
+use Illuminate\Support\Facades\Auth;
+use Carbon\Carbon;
+
+class Earhquakescontroller extends Controller
```

Comment:
Please rename your controller class name then make your file name the same as your class name.

Note:
Use capital letter to delimit words. 
For example: `UserController`, you will know immediately that this controller is for the user since it's been delimited by the capital letters.